src/bpftrace.cpp

```diff
@@ -38,6 +38,7 @@ namespace bpftrace {
 
 DebugLevel bt_debug = DebugLevel::kNone;
 bool bt_verbose = false;
+bool bt_sigint_called = false;
```

Comment:
Not sure about C++ semantics, maybe it's different, but should this be `volatile sig_atomic_t` like it's needed in C?